src/pages/tracking/components/calendar/Calendar.tsx

```diff
@@ -0,0 +1,143 @@
+    { skinStatus: "CAUTION", createAt: "2025-09-26" },
+    { skinStatus: "DANGER", createAt: "2025-09-27" },
+  ],
+};
+
+interface MonthlySkinStatusType {
+  skinStatus: SkinStatusType;
+  createAt: string;
+}
+
+interface CustomPickersDayProps extends PickersDayProps {
+  MonthlySkinStatus?: MonthlySkinStatusType[];
+}
+
+// skinStatus == 양호시 스타일
+const GoodDay = MUIstyled(PickersDay)(({ theme }) => ({
+  "&.MuiPickersDay-root.Mui-selected": {
+    backgroundColor: theme.palette.primary.light,
+  },
+}));
+
+// skinStatus == 주의시 스타일
+const CautionDay = MUIstyled(PickersDay)(({ theme }) => ({
+  "&.MuiPickersDay-root.Mui-selected": {
+    backgroundColor: theme.palette.primary.main,
+  },
+}));
+
+// skinStatus == 위험시 스타일
+const DangerDay = MUIstyled(PickersDay)(({ theme }) => ({
+  "&.MuiPickersDay-root.Mui-selected": {
+    backgroundColor: theme.palette.primary.dark,
+  },
+}));
+
+// 해당 하는 날짜가 아닐시 비활성화 스타일
+const DisabledDay = MUIstyled(PickersDay)(() => ({
+  backgroundColor: "white !important",
+  color: "black !important",
+  pointerEvents: "none", // 클릭 불가
+  "&:hover": {
+    backgroundColor: "white !important", // hover 효과 제거
+  },
+  "&.MuiPickersDay-today": {
+    border: "none", // 외각선 비활성화
+  },
+}));
+
+const Calendar = () => {
+  const [selectedDate, setSelectedDate] = useState<Dayjs | null>(dayjs());
+
+  const ServerDay = (props: CustomPickersDayProps) => {
+    const { MonthlySkinStatus = [], day, outsideCurrentMonth, ...other } = props;
+
+    // MonthlySkinStatus에 들어있지않은 날짜는 isSelected 비부여
+    const dateStr = day.format("YYYY-MM-DD");
+    const dayData = MonthlySkinStatus.find((d) => d.createAt == dateStr);
+    const isSelected = !props.outsideCurrentMonth && !!dayData;
+
+    // 활성화된 날짜 클릭시 호출되는 이벤트 리스너
+    const handleClick = () => {
+      if (dayData) {
+        console.log("클릭한 날짜:", day.format("YYYY-MM-DD"), "상태:", dayData.skinStatus);
+      }
+    };
+
+    // skinStatus에 따라서 배경색 변경
+    if (dayData) {
+      switch (dayData.skinStatus) {
+        case "GOOD":
+          return (
+            <GoodDay
+              {...other}
+              outsideCurrentMonth={outsideCurrentMonth}
+              day={day}
+              selected={isSelected}
+              onClick={handleClick}
+            />
+          );
+        case "CAUTION":
+          return (
+            <CautionDay
+              {...other}
+              outsideCurrentMonth={outsideCurrentMonth}
+              day={day}
+              selected={isSelected}
+              onClick={handleClick}
+            />
+          );
+        case "DANGER":
+          return (
+            <DangerDay
+              {...other}
+              outsideCurrentMonth={outsideCurrentMonth}
+              day={day}
+              selected={isSelected}
+              onClick={handleClick}
+            />
+          );
+      }
+    }
+    return <DisabledDay {...other} outsideCurrentMonth={outsideCurrentMonth} day={day} disabled />;
+  };
```

Comment:
P2) 현재 스타일만 다르고 모두 반복되는 구조로 보이는데, 

```typescript
const statusColorMap: Record<SkinStatusType, string> = {
  GOOD: muiTheme.palette.primary.light,
  CAUTION: muiTheme.palette.primary.main,
  DANGER: muiTheme.palette.primary.dark,
};
```

이런 형태로 상단 스타일 관련 코드를 매핑을 해 주고 

```suggestion
if (dayData) {
  return (
    <CustomDay
      {...other}
      outsideCurrentMonth={outsideCurrentMonth}
      day={day}
      selected={isSelected}
      bgColor={statusColorMap[dayData.skinStatus]}
      onClick={handleClick}
    />
  );
}

```

형태로 바꿔주면 좋을 것 같아요!